rosdep/python.yaml

```diff
@@ -2493,6 +2493,20 @@ python-levenshtein:
   fedora: [python-Levenshtein]
   gentoo: [dev-python/python-levenshtein]
   ubuntu: [python-levenshtein]
+python-pylibdmtx-pip: &migrate_eol_2025_04_30_python3_pylibdmtx_pip
+  debian:
```

Comment:
It looks like this package is available from the system as of bullseye: https://packages.debian.org/search?keywords=pylibdmtx&searchon=names&suite=all&section=all .  So let's add the system package there, and a pip fallback for everything else.

Reply:
We're part of the way there.  Since this key is going to have system packages going forward, we should have *only* a `python3-pylibdmtx` key.  For Ubuntu on Focal and earlier, this key uses `pip`, while for anything later it uses the system package.  See https://github.com/ros/rosdistro/blob/9d527af39369fc9665b563b31008a52085c89505/rosdep/python.yaml#L560-L570 for an example of what I am talking about.